### PR TITLE
Improved documentation on libos/net-next-sim + mptcp installation

### PR DESCRIPTION
**Pull request by teto**

### CODE REVIEW

File: doc/source/dce-user-kernel.rst [hunk 1]
```diff
@@ -49,11 +68,128 @@ If everything is fine, you will see **liblinux.so** linked to **libsim-linuxv.y.
 
     make menuconfig ARCH=sim
 
-.. figure:: images/dce-make-menuconfig.png
-   :scale: 70 %
+
+2.2 Build an MPTCP kernel with *net-next-sim*
+--------------------------------------------------------------------
+
+DISCLAIMER: This is a transcript of `Hajime's guide https://plus.google.com/+HajimeTazaki/posts/1QUmR3n3vNA`  updated on a best effort basis. Hence it may be possible to compile newer versions, in which sae patches are welcome.
```
Comment (thehajime) [inline doc/source/dce-user-kernel.rst]: s/sae/case/

File: doc/source/dce-user-kernel.rst [hunk 2]
```diff
@@ -49,11 +68,128 @@ If everything is fine, you will see **liblinux.so** linked to **libsim-linuxv.y.
...
+Build steps for DCE integration is also available as a script in *utils/mptcp-build.sh*.
```
Comment (thehajime) [inline doc/source/dce-user-kernel.rst]: I couldn't find where ```utils/mptcp-build.sh``` is available...

**Comment by thehajime** [comment]
Thanks @teto for the great work !
I'm not good at documenting but have been realized these kind of information is so much valuable.

I put a couple of minor comments so that you can improve the document.

Thanks again !

**Comment by teto** [comment]
I though utils/mptcp-build.sh was part of DCE and did not add it to the list of tracked files.
It's hard to provide a script since it depends on the user setup but I believe it's still handy if people want to copy/paste the commands (I know I like these kinds of reminders).

**Pull request merged**